workspace.jsonc

```diff
@@ -290,6 +290,7 @@
         "terser-webpack-plugin": "5.1.1",
         "tippy.js": "6.2.7",
         "typedoc": "0.18.0",
+        "typescript": "^4.2.3",
```

Comment:
typescript is used by the legacy from package.json, so instead of adding it here, you should change it in the package.json (so we won't have duplicate conflicted definition for this).
Also, I'm not sure if we want it with ^ or not. let's talk about it

Reply:
You're right, it should use `^`